frontend/components/MainPageV1/Home/Overview/AgentInfo/AgentActivity/index.tsx

```diff
@@ -20,7 +20,6 @@ const RoundInfoContainer = styled.div`
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  width: 500px;
```

Comment:
quickly fixed this as it wasn't as per figma (the text was centered but it shouldn't
<img width="813" height="388" alt="Screenshot 2025-10-15 at 13 30 36" src="https://github.com/user-attachments/assets/4de61177-a1a3-41ee-9c0f-1fa9cd455976" />